tools/logparser2.py

```diff
@@ -111,6 +111,36 @@ def _meta(log):
             yield sensor, os.path.basename(filepath)
 
 
+def _merge(streams):
+    vector = []
+    valid = []
```

Comment:
I do not like the `valid` here ... it is rather `active` or `has_next`?? I would expect `valid` to by Boolean (True/False)

Reply:
Renamed to `active`.